doc/nrf/create_application.rst

```diff
@@ -0,0 +1,216 @@
+.. _create_application:
+
+Create an application
+#####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The process for creating an application in the |NCS| follows Zephyr's :ref:`Application Development principles <zephyr:application>` and the |NCS|'s :ref:`dev-model`.
+The following sections provide basic information based on these two sources.
+
+.. _create_application_structure:
+
+Application structure
+*********************
+
+Here are the files that most of the Zephyr and |NCS| applications include in its :file:`<app>` directory:
+
+.. code-block:: none
+
+   <app>
```

Comment:
Should add a `VERSION` file too (added an upstream PR to document it: https://github.com/zephyrproject-rtos/zephyr/issues/61060)

Reply:
Is there a PR or there's only an issue for the time being?

Reply:
Have created a PR which contains details on how to use it

Reply:
It has now been merged, should appear on the upstream documentation, can do a cherry-pick to sdk-zephyr if needed?

Reply:
Related PR from Jamie: https://github.com/nrfconnect/sdk-zephyr/pull/1267